src/samples/UWP/Uno.Themes.Samples.Shared/Helpers/VisualTreeHelperEx.cs

```diff
@@ -291,10 +291,10 @@ IEnumerable<string> GetDetails()
 			if (x is FrameworkElement fe)
 			{
 				yield return $"Actual={fe.ActualWidth}x{fe.ActualHeight}";
-				if (fe.TransformToVisual(Window.Current.Content).TransformPoint(default) is var absPos)
-				{
-					yield return $"AbsPos={absPos.X},{absPos.Y}";
-				}
+				//if (fe.TransformToVisual(Window.Current.Content).TransformPoint(default) is var absPos)
+				//{
+				//	yield return $"AbsPos={absPos.X},{absPos.Y}";
+				//}
```

Comment:
I this normal to stay commented @Kunal22shah, @Xiaoy312 ?

Reply:
yeah
we still need it, but it is not working now
but i dont want to spent time fixing it now

Reply:
@Xiaoy312 can you add just a little comment that this still needs additional work with a link to a related issue, so it will be easier to understand the context of this change, please?

Reply:
it wont be necessary. this is just a debug method